src/Numerical/Spectral/Linearize.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function Linearize.
+
+#pragma once
+
+#include <cstddef>
+
+class DataVector;
+template <size_t>
+class Index;
+template <size_t Dim>
```

Comment:
remove the type alias and change the `Mesh` below to `Index` with the argument name `extents`